include/swift/Reflection/TypeLowering.h

```diff
@@ -239,6 +239,14 @@ class EnumTypeInfo : public TypeInfo {
     return std::count_if(Cases.begin(), Cases.end(),
                          [](const FieldInfo &Case){return Case.TR != 0;});
   }
+  unsigned getNumNonEmptyPayloadCases() const {
```

Comment:
Looks like this is only used in asserts, does this need any magic to avoid an unused-function warning in a no-asserts build?